assets/style.css

```diff
@@ -1,5 +1,89 @@
+/* stylelint-disable color-hex-case */
 /* stylelint-disable indentation */
 /* stylelint-disable no-missing-end-of-source-newline */
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap');
+
 body {
         background-color: white;
+}
+
+#header_menu {
+       display: flex;
+       width: 100%;
+       list-style-type: none;
+}
+
+#header_menu .welcome_text {
+        width: 90px;
+        height: 28px;
+        font-family: 'Inter', sans-serif;
+        font-style: normal;
+        font-weight: bolder;
+        font-size: 20px;
+        line-height: 28px;
+        flex: 3;
+        order: 1;
+}
+
+.toggleNav {
+        left: 16.67%;
+        right: 16.67%;
+        top: 50%;
+        bottom: 50%;
+        flex: 1;
+        text-align: right;
+        order: 2;
+        flex-grow: 0;
+}
+
+#presentation_text {
+        top: 156px;
+        font-family: 'Inter', sans-serif;
+        font-weight: 700;
+        font-size: 40px;
+        line-height: 52px;
+        text-align: center;
+}
```

Comment:
- [ ] [OPTIONAL] The design will look great if you bold the headline. I highly recommend it!

## Current:
![image](https://user-images.githubusercontent.com/92215878/191731725-a9e95450-13a7-46a7-9b43-16dffe2e61ce.png)

## Figma:
![image](https://user-images.githubusercontent.com/92215878/191731638-c7ee9b2b-98b1-4e19-8046-2ff04e6f79b0.png)